### PR TITLE
Add victoria-lo

### LINKED ISSUE #590: Victoria Lo
image

### PR DESCRIPTION
**Pull request by victoria-lo**
Resolve #590

### CODE REVIEW

**Review by schmelto — CHANGES_REQUESTED** [review]
Please add yourself at the end of the data.json file

**Review by Panquesito7 — COMMENTED** [review]
I think you should re-open the #590 issue. It should be closed once this PR is merged.

**Comment by victoria-lo** [comment]
I have reopened the issue and add myself at the end of data.json. Thanks :)

File: data.json [hunk 1]
```diff
@@ -1114,5 +1114,11 @@
     "githubUsername": "AnkitSharma-007",
     "imageUrl": "https://user-images.githubusercontent.com/33789321/124962632-7cb79e00-e03c-11eb-9a5c-d7699fa3a6b2.png",
     "issueNumber": 580
+  },
+  {
+    "name": "Victoria Lo",
+    "githubUsername": "victoria-lo",
+    "imageUrl":"https://user-images.githubusercontent.com/47699978/125807703-6d8c6adf-d642-42f2-b1bf-51f6a32c0a84.png"
```
Comment (schmelto) [inline data.json:1121]:
```suggestion
    "imageUrl":"https://user-images.githubusercontent.com/47699978/125807703-6d8c6adf-d642-42f2-b1bf-51f6a32c0a84.png", 
```

**Review by schmelto — CHANGES_REQUESTED** [review]

File: data.json [hunk 2]
```diff
@@ -1115,6 +1115,12 @@
     "imageUrl": "https://user-images.githubusercontent.com/33789321/124962632-7cb79e00-e03c-11eb-9a5c-d7699fa3a6b2.png",
     "issueNumber": 580
   },
+  {
+    "name": "Victoria Lo",
+    "githubUsername": "victoria-lo",
+    "imageUrl":"https://user-images.githubusercontent.com/47699978/125807703-6d8c6adf-d642-42f2-b1bf-51f6a32c0a84.png",
```
Comment (rahmanharoon) [inline data.json:1121]:
```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/47699978/125807703-6d8c6adf-d642-42f2-b1bf-51f6a32c0a84.png", 
```

**Review by rahmanharoon — CHANGES_REQUESTED** [review]

**Review by schmelto — APPROVED** [review]
lgtm

**Comment by schmelto** [comment]
Thanks for your contribution and collaboration 🚀

**Pull request merged**